internal/contributebot/process.go

```diff
@@ -27,11 +27,13 @@ import (
 const (
 	inProgressLabel            = "in progress"
 	issueTitleComment          = "Please edit the title of this issue with the name of the affected package, followed by a colon, followed by a short summary of the issue. Example: `blob/gcsblob: not blobby enough`."
+	pullRequestTitleComment    = "Please edit the title of this pull request with the name of the affected package, followed by a colon, followed by a short summary of the change. Example: `blob/gcsblob: improve comments`."
```

Comment:
... with the name of the affected package, or "all", followed by a colon,...

Reply:
Done.